src/pkg/projects/Microsoft.Net.UWPCoreRuntimeSdk/contents/identity/Microsoft.Net.CoreRuntime.targets

```diff
@@ -0,0 +1,277 @@
+      <Output TaskParameter="InstalledSDKs" ItemName="InstalledSDKLocations"/>
+    </GetInstalledSDKLocations>
+
+
+    <ItemGroup>
+      <DependencySDKs Include="$(VCLibs14SDKName)" />  <!--UWPShim is built against dev14 CRT-->
+    </ItemGroup>
+
+    <ResolveSDKReference
+      SDKReferences="@(DependencySDKs)"
+      TargetPlatformVersion="$(TargetPlatformVersion)"
+      TargetPlatformIdentifier="$(TargetPlatformIdentifier)"
+      TargetedSDKConfiguration="$(TargetedSDKConfiguration)"
+      TargetedSDKArchitecture="$(TargetedSDKArchitecture)"
+      ProjectName="$(ProjectName)"
+      InstalledSDKs ="@(InstalledSDKLocations)"
+    >
+      <Output TaskParameter="ResolvedSDKReferences" ItemName="ResolvedDependencySDKs"/>
+    </ResolveSDKReference>
+
+    <ItemGroup>
+      <CoreRuntimeSDK Include="@(ResolvedDependencySDKs)" Condition="'%(ResolvedDependencySDKs.SDKName)' == '$(CoreRuntimeSDKName)'" />
+    </ItemGroup>
+
+    <GetFrameworkSdkPackages FrameworkSdkReferences="@(ResolvedDependencySDKs)"
+        TargetPlatformIdentifier="$(TargetPlatformIdentifier)">
+      <Output TaskParameter="FrameworkSdkPackages" ItemName="FrameworkPackageDependencies" />
+    </GetFrameworkSdkPackages>
+
+
+    <ItemGroup>
+      <FrameworkPackagesForTargetArchitecture Include="@(FrameworkPackageDependencies)" Condition="'%(Architecture)' == '$(_PlatformTargetForCoreRuntime)'" />
+    </ItemGroup>
+
+    <PropertyGroup>
+      <!-- Name of manifest file is predefined -->
+      <_AppxManifestXmlFileName>AppxManifest.xml</_AppxManifestXmlFileName>
+      <!-- Expected location of result manifest file where it supposed to be dropped by WireUpCoreRuntime -->
+      <_TransformedAppxManifestXmlFile>$(_WireUpCoreRuntimeOutputPath)$(_AppxManifestXmlFileName)</_TransformedAppxManifestXmlFile>
+
+      <CoreRuntimeSDKLocation Condition="'$(PlatformTarget)' == 'x86'">$(CoreRuntimeSDKRootX86)</CoreRuntimeSDKLocation>
+      <CoreRuntimeSDKLocation Condition="'$(PlatformTarget)' == 'x64'">$(CoreRuntimeSDKRootX64)</CoreRuntimeSDKLocation>
+      <CoreRuntimeSDKLocation Condition="'$(PlatformTarget)' == 'arm'">$(CoreRuntimeSDKRootArm)</CoreRuntimeSDKLocation>
+    </PropertyGroup>
+  </Target>
+
+  <Target Name="_GetLibrariesToGeneratePrisForUWPApps"
+          BeforeTargets="_GeneratePrisForPortableLibraries"
+          Condition="'$(TargetPlatformIdentifier)'=='UAP'">
+
+        <ItemGroup>
+          <_FullAppLibrariesUnfiltered Include="@(PackagingOutputs)" Condition="'%(Extension)' == '.dll'" />
+        </ItemGroup>
+
+        <ConvertToAbsolutePath Paths="$(IntermediateOutputPath)">
+          <Output TaskParameter="AbsolutePaths" PropertyName="ResWOutputFullPath"/>
+        </ConvertToAbsolutePath>
+
+        <PropertyGroup Condition="'$(SkipMergingFrameworkResources)' == ''">
+          <!--
+            When building ProjectN apps in retail mode we need to skip including the framework resources in the app because we optimize the exception stack
```

Comment:
Not sure if it's right to have branch names like `ProjectN` here in a comment on a target that will be imported by all UWP projects.